cassandra-trunk/Dockerfile.ubi8

```diff
@@ -42,7 +44,7 @@ RUN set -x \
     && rm -rf /var/lib/apt/lists/* \
     && apt-get update \
     && apt-get install -y --no-install-recommends git ant ant-optional make maven \
-    && git clone -b trunk --single-branch https://github.com/apache/cassandra.git \
+    && git clone -b ${BRANCH} --single-branch ${REPO} \
```

Comment:
So this will work as long as the branch you pick is based on a 5.0 version of Cassandra, and not a Release branch (i.e. the build artifact will be `5.0-SNAPSHOT`. If not, then you will need to handle:

1. https://github.com/yukim/management-api-for-apache-cassandra/blob/feature/cassandra-repo-arg/cassandra-trunk/Dockerfile.ubi8#L53

As the build artifact is assumed to be `5.0-SNAPSHOT`

2. https://github.com/yukim/management-api-for-apache-cassandra/blob/feature/cassandra-repo-arg/cassandra-trunk/Dockerfile.ubi8#L90-L97

You also have to adjust which artifacts of Management API are removed, kept and sym-linked in the section above. There are different artifacts built for different versions of Cassandra and things won't work if you include the wrong one.

3. https://github.com/yukim/management-api-for-apache-cassandra/blob/feature/cassandra-repo-arg/cassandra-trunk/Dockerfile.ubi8#L103

The `CDC_AGENT_EDITION` variable also needs to changed, depending on which version of Cassandra.

Having said all this, your changes should work as long as the branch is a 5.0-SNAPSHOT version. Seeing as how this Dockerfile is in the `cassandra-trunk` folder, I think that is a safe assumption for now.

I can approve this PR as-is, with all of the above caveats noted. However, it would be nice if you could figure out a way to deduce the Cassandra version so it's not hardcoded to `5.0-SNAPSHOT` here: https://github.com/yukim/management-api-for-apache-cassandra/blob/feature/cassandra-repo-arg/cassandra-trunk/Dockerfile.ubi8#L53

Let me know if you are willing to try this and I can wait to approve this PR then. If not, I think I can approve it now and this can be addressed later.